graphs/minimum_spanning_tree_prims.py

```diff
@@ -125,9 +127,9 @@ def prisms_algorithm(l):
     # < --------- Prims Algorithm --------- >
     n = int(input("Enter number of vertices: ").strip())
     e = int(input("Enter number of edges: ").strip())
```

Comment:
Do we gather this data and then not use it?

Reply:
Yes, `n` wasn't used, but `e` is used